### PR TITLE
ADDED A PROJECT: Real Time INTERNATIONAL SPACE STATION Tracker with Astronaut On Board

### PR DESCRIPTION
**Pull request by NirbheekKumar**
Project Overview: Real-time tracking of the International Space Station (ISS) with astronaut information.

Functionality:
Real-Time Tracking: Provides up-to-date location data of the ISS in latitude and longitude
Astronaut Information: Displays names of astronauts currently aboard the ISS.

Open APIs Used:
ISS Location API: Retrieves real-time positional data of the ISS.
Astronauts API: Fetches current crew information and astronaut names.

Features:
Interactive Map: Visual representation of the ISS’s current location on a map.
Live Updates: Continuous refresh of ISS position and astronaut details.
User Interface: User-friendly interface to view ISS location and astronaut information.

Technical Stack:
Pyhton and APIs: Integration of Open APIs for real-time data.

Use Cases:
Educational: Enhances learning about space and the ISS.
Public Engagement: Provides real-time space station tracking for enthusiasts.
Research: Supports research in space exploration and astronaut activities.

STEPS TO IMPLEMENT FOR OTHER USERS TO RUN LOCALLY:-

UPDATE THE main.py FILE:-

display.bgpic("map.gif") # display.bgpic("ADD PATH OF map.gif AS PER YOUR DIRECTORY")
display.register_shape("iss.gif") # display.register_shape("ADD PATH OF iss.gif AS PER YOUR DIRECTORY")
iss.shape("iss.gif") # iss.shape("ADD PATH OF iss.gif AS PER YOUR DIRECTORY")

Basically adding the path of images of map.gif and iss.gif in main.py file as per your directory

WORKING IMAGES OF THE PROGRAM

Screenshot 2024-07-14 045606
Screenshot 2024-07-14 045633
Screenshot 2024-07-14 045822
Screenshot 2024-07-14 045532

### CODE REVIEW

**Comment by NirbheekKumar** [comment]
I have opened an issue #906  for adding a health care project. Kindly review it.

**Comment by SyedImtiyaz-1** [comment]
Hello @NirbheekKumar ,
Can you please add data in the `.json` file.

Will merge this with level 3.

**Comment by NirbheekKumar** [comment]
I have added data in.json file of ISS

**Comment by NirbheekKumar** [comment]
I have given two project actually : one is ISS and another is HEALTH GAUGE,: the health_monitor is added as a sub module

**Comment by NirbheekKumar** [comment]
> There is a mistake in your codebase due to json file.
> 
> * You have to add it in the given json format, there is no need to create one.
> * Check backend/projects/.json
> 
> `Under Review !`

so i have to add .json in your backend/ projects.json?

**Comment by NirbheekKumar** [comment]
DO I HAVE TO ADD DATA TO THIS JSON FILE OF YOURS ?

Screenshot 2024-07-17 235928

**Comment by NirbheekKumar** [comment]
I HAVE ADDED THE DATA IN backend/public/projects.json

Screenshot 2024-07-18 002115

**Comment by NirbheekKumar** [comment]
HAVE UPDATED backend/public/projects.json inthe given format

**Comment by NirbheekKumar** [comment]
I have added and updated the .json file in backend/public/projects.json

**Comment by NirbheekKumar** [comment]
I have updated all the required things in json file and uploaded links of logo and images. Please check and merge it.

ThankYou

**Comment by NirbheekKumar** [comment]
I have updated the json file as per the formate, please check and verify.

**Comment by NirbheekKumar** [comment]
Any conflicts for this ?
Please let me know

**Comment by NirbheekKumar** [comment]
Please merge this as well

Thank you

**Comment by NirbheekKumar** [comment]
Do check and verify this updated files 
MERGE THIS PR

ThankYou

**Comment by NirbheekKumar** [comment]
HAVE UPDATE IN main.py FILE

STEPS TO IMPLEMENT FOR OTHER USERS:-

display.bgpic("map.gif") # display.bgpic("ADD PATH OF map.gif AS PER YOUR DIRECTORY")

display.register_shape("iss.gif") # display.register_shape("ADD PATH OF iss.gif AS PER YOUR DIRECTORY")

iss.shape("iss.gif") # iss.shape("ADD PATH OF iss.gif AS PER YOUR DIRECTORY")

**Comment by NirbheekKumar** [comment]
Please check and verify this project and merge the pr

ThankYou
